cpp/src/io/utilities/parsing_utils.cuh

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <vector>
+
+#include "cudf.h"
+
+using ll_uint_t = unsigned long long int;
```

Comment:
Why the change in type name? I'd rather we keep the higher-level logical naming like pos_offset_type or similar (like gdf_size_type). Or we be explicit in type size like uint64_t.

Reply:
Used the alias because CUDA atomic operations are very picky about the parameter type. However, with the recent code refactoring, this is no longer needed. Changed to uint64_t.